src/main/java/org/benetech/servicenet/domain/OrganizationMatch.java

```diff
@@ -39,6 +39,15 @@
     @Column(name = "dismiss_date")
     private ZonedDateTime dismissDate;
 
+    @Column(name = "hidden")
+    private Boolean hidden = false;
+
+    @ManyToOne
+    private User hiddenBy;
```

Comment:
I think you can specify the join column

Reply:
What's the point of it? You didn't do this for dismissedBy.

Reply:
it's the same like specifying the name of a column, in the code I see some of the relations have the join column and some don't have it
you can just leave it like it is